### DISCUSSION TITLE
Configure email relay via Office 365 (Windows)

### DISCUSSION THREAD

**Discussion by itamarka9** [requarks/wiki#6890] [Guides]
I installed Wiki.js on a windows server.
I want to configure the SMTP to relay the messages via O365.
When I test my settings, it fails with ssl3_get_recored: wrong version.
How do I fix it?
TLS is enabled and Verify SSL is enabled too. 
If I disable SSL verification, I get the same error.

Thanks

**Comment by tim-elmer** [comment]
Have you verified that this is even supported? Last time I checked, Microsoft doesn't offer email relay as a service. You may want to look at options like MailTrap, Mailgun, AWS SES, etc.
